app/scripts/modules/google/src/securityGroup/details/securityGroupDetail.html

```diff
@@ -144,6 +144,22 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>
       </ul>
       <div ng-if="!securityGroup.sourceTags.length">None</div>
     </collapsible-section>
+    <collapsible-section heading="Target Service Accounts ({{securityGroup.targetServiceAccounts.length || 0}})">
```

Comment:
Is there ever a case where both tags and service accounts are present on a security group? My understanding is that it's an either-or or not present for 'default' security groups. Would it be better to have some conditional rendering instead of showing at least one empty accordion?

Reply:
Discussed with @maggieneterval and actually keeping both present (even if they are empty) is clearer from a UX perspective.

Reply:
@jtk54 and I just chatted offline and decided to hold off on this conditional rendering for now in case it causes more confusion than it's worth.

Thanks for the PR, @seanpeters86!